### PR TITLE
Equalize margins between properties on details tab

### PR DESCRIPTION
**Pull request by st3iny**
## Before
Screenshot 2021-10-01 at 14-53-25 Woche 39 aus 2021 - Kalender - Nextcloud

## After
Screenshot 2021-10-01 at 14-56-39 Woche 39 aus 2021 - Kalender - Nextcloud

### CODE REVIEW

File: css/app-sidebar.scss [hunk 1]
```diff
@@ -562,6 +563,9 @@
 
 			textarea {
 				max-height: calc(100vh - 600px);
+
+				// Revert weird margin from global server style sheet and even out margins between properties.
+				margin: 0 0 -2.8px;
```
Comment (szaimen) [inline css/app-sidebar.scss:568]: This looks strange to me

**Review by szaimen — APPROVED** [review]
Lgtm apart from small detail

File: css/app-sidebar.scss (hunk 1 above)
Comment (st3iny) [reply css/app-sidebar.scss]: Yeah, I don't like it either :laughing: This was an evil hack to eliminate the extra margin below a text area but there is a proper way to do it: https://stackoverflow.com/a/7144960

**Comment by st3iny** [comment]
> The backport to stable2.3 failed. Please do this backport manually.

Let's not then. It was worth a try though.

**Pull request merged**